logsearchapi/main.go

```diff
@@ -26,6 +26,8 @@ import (
 	"os"
 	"strconv"
 
+	_ "github.com/lib/pq"
```

Comment:
maybe the linter will complaint unless you add a comment

Reply:
It complained that it needs to be in main. So i moved it here. This should be fine.